src/auro-alaska.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable one-var */
+// Copyright (c) 2020 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+import { html, css } from "lit-element";
+import { classMap } from 'lit-html/directives/class-map';
+import BaseIcon from "./baseIcon";
+
+import as400 from '@alaskaairux/icons/dist/restricted/AS-400_es6.js';
+import as300 from '@alaskaairux/icons/dist/restricted/AS-300_es6.js';
+import as200 from '@alaskaairux/icons/dist/restricted/AS-200_es6.js';
+import as100 from '@alaskaairux/icons/dist/restricted/AS-100_es6.js';
+import official400 from '@alaskaairux/icons/dist/restricted/AS-tagline-400_es6.js';
+import official300 from '@alaskaairux/icons/dist/restricted/AS-tagline-300_es6.js';
+import official200 from '@alaskaairux/icons/dist/restricted/AS-tagline-200_es6.js';
+import official100 from '@alaskaairux/icons/dist/restricted/AS-tagline-100_es6.js';
+
+// Import touch detection lib
+// import "focus-visible/dist/focus-visible.min.js";
+import styleCss from "./alaskaStyle-css.js";
+
+// See https://git.io/JJ6SJ for "How to document your components using JSDoc"
+/**
+ * auro-alaska provides users a way to use the Alaska Airline logos.
+ *
+ * @attr {Boolean} official - Set value for alaska airlines logo with official tagline
+ * @slot - Hidden from visibility, used for a11y if icon description is needed
+ */
+
+// build the component class
+class AuroAlaska extends BaseIcon {
+  constructor() {
+    super();
+
+    this.privateDefaults();
+  }
+
+  /**
+   * @private internal defaults
+   * @returns {void}
+   */
+  privateDefaults() {
+    this.uri = 'https://unpkg.com/@alaskaairux/icons@latest/dist';
+    this.zero = 0;
```

Comment:
I don't think this is being used anymore

Reply:
```suggestion
```

Reply:
DOH!